<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="27" passed="27" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-11-17T20:24:17Z" name="Surefire suite" finished-at="2019-11-17T20:24:17Z" duration-ms="37">
    <groups>
    </groups>
    <test started-at="2019-11-17T20:24:17Z" name="Surefire test" finished-at="2019-11-17T20:24:17Z" duration-ms="37">
      <class name="com.bieganski.board.BoardParserTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.bieganski.board.BoardParserTest@4e718207]" started-at="2019-11-17T20:24:17Z" name="setUp" finished-at="2019-11-17T20:24:17Z" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="parse_ThrowsExceptionWhenNull()[pri:0, instance:com.bieganski.board.BoardParserTest@4e718207]" started-at="2019-11-17T20:24:17Z" name="parse_ThrowsExceptionWhenNull" finished-at="2019-11-17T20:24:17Z" duration-ms="2" status="PASS">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[List of text lines can't be null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: List of text lines can't be null
at com.bieganski.board.BoardParser.parse(BoardParser.java:23)
at com.bieganski.board.BoardParserTest.parse_ThrowsExceptionWhenNull(BoardParserTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- parse_ThrowsExceptionWhenNull -->
        <test-method signature="parse_WrongLettersInFile()[pri:0, instance:com.bieganski.board.BoardParserTest@4e718207]" started-at="2019-11-17T20:24:17Z" name="parse_WrongLettersInFile" finished-at="2019-11-17T20:24:17Z" duration-ms="1" status="PASS">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Color can be only W-White or B-Black]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Color can be only W-White or B-Black
at com.bieganski.board.BoardParser.parse(BoardParser.java:34)
at com.bieganski.board.BoardParserTest.parse_WrongLettersInFile(BoardParserTest.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- parse_WrongLettersInFile -->
      </class> <!-- com.bieganski.board.BoardParserTest -->
      <class name="com.bieganski.figures.KingFigureTest">
        <test-method signature="checkCollision_CanNotReturnNull()[pri:0, instance:com.bieganski.figures.KingFigureTest@c81cdd1]" started-at="2019-11-17T20:24:17Z" name="checkCollision_CanNotReturnNull" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCollision_CanNotReturnNull -->
      </class> <!-- com.bieganski.figures.KingFigureTest -->
      <class name="com.bieganski.board.BoardReaderTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.bieganski.board.BoardReaderTest@1d371b2d]" started-at="2019-11-17T20:24:17Z" name="setUp" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="readFileToList_LoadProperFile()[pri:0, instance:com.bieganski.board.BoardReaderTest@1d371b2d]" started-at="2019-11-17T20:24:17Z" name="readFileToList_LoadProperFile" finished-at="2019-11-17T20:24:17Z" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readFileToList_LoadProperFile -->
        <test-method signature="readFileToList_ThrowsExceptionWhenArgumentIsNull()[pri:0, instance:com.bieganski.board.BoardReaderTest@1d371b2d]" started-at="2019-11-17T20:24:17Z" name="readFileToList_ThrowsExceptionWhenArgumentIsNull" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[File name can't be null!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: File name can't be null!
at com.bieganski.board.BoardReader.readFileToList(BoardReader.java:12)
at com.bieganski.board.BoardReaderTest.readFileToList_ThrowsExceptionWhenArgumentIsNull(BoardReaderTest.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readFileToList_ThrowsExceptionWhenArgumentIsNull -->
        <test-method signature="readFileToList_ThrowsExceptionWhenFileNameIsBad()[pri:0, instance:com.bieganski.board.BoardReaderTest@1d371b2d]" started-at="2019-11-17T20:24:17Z" name="readFileToList_ThrowsExceptionWhenFileNameIsBad" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[There is no file like: abc]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: There is no file like: abc
at com.bieganski.board.BoardReader.readFileToList(BoardReader.java:29)
at com.bieganski.board.BoardReaderTest.readFileToList_ThrowsExceptionWhenFileNameIsBad(BoardReaderTest.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readFileToList_ThrowsExceptionWhenFileNameIsBad -->
        <test-method signature="redFileToList_ShouldNotBeNull()[pri:0, instance:com.bieganski.board.BoardReaderTest@1d371b2d]" started-at="2019-11-17T20:24:17Z" name="redFileToList_ShouldNotBeNull" finished-at="2019-11-17T20:24:17Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- redFileToList_ShouldNotBeNull -->
      </class> <!-- com.bieganski.board.BoardReaderTest -->
      <class name="com.bieganski.figures.FigureTest">
        <test-method signature="toString_CanNotReturnNull()[pri:0, instance:com.bieganski.figures.FigureTest@43738a82]" started-at="2019-11-17T20:24:17Z" name="toString_CanNotReturnNull" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- toString_CanNotReturnNull -->
      </class> <!-- com.bieganski.figures.FigureTest -->
      <class name="com.bieganski.board.CharConverterTest">
        <test-method signature="convertCharNumberToInt_FailWhenArgumentIsNotNumber()[pri:0, instance:com.bieganski.board.CharConverterTest@13eb8acf]" started-at="2019-11-17T20:24:17Z" name="convertCharNumberToInt_FailWhenArgumentIsNotNumber" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Argument isn't number]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Argument isn't number
at com.bieganski.board.CharConverter.convertCharNumberToInt(CharConverter.java:18)
at com.bieganski.board.CharConverterTest.convertCharNumberToInt_FailWhenArgumentIsNotNumber(CharConverterTest.java:11)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- convertCharNumberToInt_FailWhenArgumentIsNotNumber -->
        <test-method signature="convertCharNumberToInt_ProperValue()[pri:0, instance:com.bieganski.board.CharConverterTest@13eb8acf]" started-at="2019-11-17T20:24:17Z" name="convertCharNumberToInt_ProperValue" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- convertCharNumberToInt_ProperValue -->
        <test-method signature="convertCharNumberToInt_WrongValue()[pri:0, instance:com.bieganski.board.CharConverterTest@13eb8acf]" started-at="2019-11-17T20:24:17Z" name="convertCharNumberToInt_WrongValue" finished-at="2019-11-17T20:24:17Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- convertCharNumberToInt_WrongValue -->
        <test-method signature="getIndexOfLetterInAlphabet_ProperValue()[pri:0, instance:com.bieganski.board.CharConverterTest@13eb8acf]" started-at="2019-11-17T20:24:17Z" name="getIndexOfLetterInAlphabet_ProperValue" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIndexOfLetterInAlphabet_ProperValue -->
        <test-method signature="getIndexOfLetterInAlphabet_WrongValue()[pri:0, instance:com.bieganski.board.CharConverterTest@13eb8acf]" started-at="2019-11-17T20:24:17Z" name="getIndexOfLetterInAlphabet_WrongValue" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIndexOfLetterInAlphabet_WrongValue -->
        <test-method signature="getIndexOfLetterInAlphabet_FailWhenArgumentIsNotLetter()[pri:0, instance:com.bieganski.board.CharConverterTest@13eb8acf]" started-at="2019-11-17T20:24:17Z" name="getIndexOfLetterInAlphabet_FailWhenArgumentIsNotLetter" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Argument isn't letter!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Argument isn't letter!
at com.bieganski.board.CharConverter.getIndexOfLetterInAlphabet(CharConverter.java:9)
at com.bieganski.board.CharConverterTest.getIndexOfLetterInAlphabet_FailWhenArgumentIsNotLetter(CharConverterTest.java:16)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIndexOfLetterInAlphabet_FailWhenArgumentIsNotLetter -->
        <test-method signature="getIndexOfLetterInAlphabet_LowerCaseShouldWork()[pri:0, instance:com.bieganski.board.CharConverterTest@13eb8acf]" started-at="2019-11-17T20:24:17Z" name="getIndexOfLetterInAlphabet_LowerCaseShouldWork" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIndexOfLetterInAlphabet_LowerCaseShouldWork -->
      </class> <!-- com.bieganski.board.CharConverterTest -->
      <class name="com.bieganski.board.BoardTest">
        <test-method signature="getFields_ShouldThrowException()[pri:0, instance:com.bieganski.board.BoardTest@543c6f6d]" started-at="2019-11-17T20:24:17Z" name="getFields_ShouldThrowException" finished-at="2019-11-17T20:24:17Z" duration-ms="1" status="PASS">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[List of figures can't be null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: List of figures can't be null
at com.bieganski.board.Board.<init>(Board.java:18)
at com.bieganski.board.BoardTest.getFields_ShouldThrowException(BoardTest.java:11)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFields_ShouldThrowException -->
      </class> <!-- com.bieganski.board.BoardTest -->
      <class name="com.bieganski.figures.KnightTest">
        <test-method signature="checkCollision_CanNotReturnNull()[pri:0, instance:com.bieganski.figures.KnightTest@1fc2b765]" started-at="2019-11-17T20:24:17Z" name="checkCollision_CanNotReturnNull" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCollision_CanNotReturnNull -->
      </class> <!-- com.bieganski.figures.KnightTest -->
      <class name="com.bieganski.board.FieldCoordinatesTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.bieganski.board.FieldCoordinatesTest@51c8530f]" started-at="2019-11-17T20:24:17Z" name="setUp" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="getFieldCoordinates_getProperColumnValue()[pri:0, instance:com.bieganski.board.FieldCoordinatesTest@51c8530f]" started-at="2019-11-17T20:24:17Z" name="getFieldCoordinates_getProperColumnValue" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFieldCoordinates_getProperColumnValue -->
        <test-method signature="getFieldCoordinates_getProperRowValue()[pri:0, instance:com.bieganski.board.FieldCoordinatesTest@51c8530f]" started-at="2019-11-17T20:24:17Z" name="getFieldCoordinates_getProperRowValue" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFieldCoordinates_getProperRowValue -->
        <test-method signature="getFieldCoordinates_getWrongRowValue()[pri:0, instance:com.bieganski.board.FieldCoordinatesTest@51c8530f]" started-at="2019-11-17T20:24:17Z" name="getFieldCoordinates_getWrongRowValue" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFieldCoordinates_getWrongRowValue -->
        <test-method signature="getFieldCoordinates_getWrongColumnValue()[pri:0, instance:com.bieganski.board.FieldCoordinatesTest@51c8530f]" started-at="2019-11-17T20:24:17Z" name="getFieldCoordinates_getWrongColumnValue" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFieldCoordinates_getWrongColumnValue -->
        <test-method signature="toString_ValueCanNotBe_Null()[pri:0, instance:com.bieganski.board.FieldCoordinatesTest@51c8530f]" started-at="2019-11-17T20:24:17Z" name="toString_ValueCanNotBe_Null" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- toString_ValueCanNotBe_Null -->
        <test-method signature="setRowValue_setProperRowValue()[pri:0, instance:com.bieganski.board.FieldCoordinatesTest@51c8530f]" started-at="2019-11-17T20:24:17Z" name="setRowValue_setProperRowValue" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setRowValue_setProperRowValue -->
        <test-method signature="setColumnValue_setProperColumnValue()[pri:0, instance:com.bieganski.board.FieldCoordinatesTest@51c8530f]" started-at="2019-11-17T20:24:17Z" name="setColumnValue_setProperColumnValue" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setColumnValue_setProperColumnValue -->
      </class> <!-- com.bieganski.board.FieldCoordinatesTest -->
      <class name="com.bieganski.board.FieldTest">
        <test-method signature="getFigureOnThisField_ShouldReturnNull()[pri:0, instance:com.bieganski.board.FieldTest@7403c468]" started-at="2019-11-17T20:24:17Z" name="getFigureOnThisField_ShouldReturnNull" finished-at="2019-11-17T20:24:17Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFigureOnThisField_ShouldReturnNull -->
        <test-method signature="toString_ValueCanNotBe_Null()[pri:0, instance:com.bieganski.board.FieldTest@7403c468]" started-at="2019-11-17T20:24:17Z" name="toString_ValueCanNotBe_Null" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- toString_ValueCanNotBe_Null -->
        <test-method signature="getFiguresThatCanAttackThisField_CanNotReturnNull()[pri:0, instance:com.bieganski.board.FieldTest@7403c468]" started-at="2019-11-17T20:24:17Z" name="getFiguresThatCanAttackThisField_CanNotReturnNull" finished-at="2019-11-17T20:24:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFiguresThatCanAttackThisField_CanNotReturnNull -->
      </class> <!-- com.bieganski.board.FieldTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
